aptos-move/aptos-release-builder/data/collection-owner.yaml

```diff
@@ -0,0 +1,14 @@
+---
+remote_endpoint: ~
+proposals:
+  - name: feature_flags
+    metadata:
+      title: ""
```

Comment:
fill in a short title for this and description, you can delete the two urls fields